packages/studio/styled-system/css/css.mjs

```diff
@@ -28,15 +28,22 @@ const context = {
   utility: {
     transform: (prop, value) => {
       const key = resolveShorthand(prop)
-      const propKey = classNames.get(key) || hypenateProperty(key)
+      const propKey = classNameByProp.get(key) || hypenateProperty(key)
       return { className: `${propKey}_${withoutSpace(value)}` }
     },
     hasShorthand: true,
     resolveShorthand: resolveShorthand,
   },
 }
 
-export const css = createCss(context)
+const cssFn = createCss(context)
+export const cssCache = new Map()
+export const css = (styles) => {
+  const classNames = cssFn(styles)
+  cssCache.set(classNames, styles)
+  console.log({ classNames, styles })
```

Comment:
Looks good. Can we remove this log line?

Reply:
done !